### DISCUSSION TITLE
EmptyShape Analog in Jolt

### DISCUSSION THREAD

**Discussion by kozakromch** [jrouwe/JoltPhysics#1120] [Q&A] [answered]
Hello, Thank you for the great library. Is there a way to create rigid body without any collision shape? 
In some case I need the body to obey constraints and forces and no need to collide with smth.  I know the inertia tensor and mass of the body. 
So I need smth like EmptyShape in bullet. Thank you in advance)

**Comment by SirLynix** [comment]
as far as I know there are no empty shapes, to support them in my engine I use an epsilon-sized sphere shape tagged as sensor.

**Comment by jrouwe** [comment, marked as answer]
Another (cheaper) option is to use a unit sphere and place the body in a collision layer (ObjectLayer) that doesn't collide with anything.
